src/MonitoringFunctions/HelperMethods.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+[assembly:InternalsVisibleTo("MonitoringFunctions.Test, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9")]
+
+namespace MonitoringFunctions
+{
+    internal static class HelperMethods
+    {
+        /// <summary>
+        /// Single instance is sufficient for the whole app.
+        /// From official docs: HttpClient is intended to be instantiated once and re-used throughout the life of an application.
+        /// https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient?view=netcore-3.1#remarks
+        /// </summary>
+        private static readonly HttpClient _httpClient = new HttpClient();
```

Comment:
this is IDisposable

Reply:
In this case, we need it until the app exits and it will be automatically disposed when that happens.

Reply:
you can make that same argument for most classes.  This being a static class, consider a ```CleanUp``` type method that essentially cleans up such resources & invalidates the instance.  Moving forward, callers will have the option to reclaim those resources during the life of the process.